### PR TITLE
python3Packages.karton-config-extractor: init at 1.0.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change

https://github.com/CERT-Polska/karton-config-extractor

incl. malduck (https://github.com/CERT-Polska/malduck)

Related to #81418

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

CC @chivay

### CODE REVIEW

File: pkgs/development/python-modules/malduck/default.nix [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    click
+    cryptography
+    pefile
+    pycryptodomex
+    pyelftools
+    typing-extensions
+    yara-python
+  ];
+
+  # Project has no tests. They will come with the next release
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/malduck/default.nix:39]:
```suggestion
  # Project has no tests. They will come with the next release
```

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 119461` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python38Packages.karton-config-extractor</li>
    <li>python38Packages.malduck</li>
    <li>python39Packages.karton-config-extractor</li>
    <li>python39Packages.malduck</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119461` at af6a2551 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built successfully:</summary>
  <ul>
    <li>python38Packages.karton-config-extractor</li>
    <li>python38Packages.malduck</li>
    <li>python39Packages.karton-config-extractor</li>
    <li>python39Packages.malduck</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `python`.
Near pkgs/development/python-modules/malduck/default.nix:10:3:
```
   |
10 | , python
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/malduck/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/malduck/default.nix]: Thanks, fixed.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 119461` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python38Packages.karton-config-extractor</li>
    <li>python38Packages.malduck</li>
    <li>python39Packages.karton-config-extractor</li>
    <li>python39Packages.malduck</li>
  </ul>
</details>

**Pull request merged**